web/package.json

```diff
@@ -25,6 +25,7 @@
     "apollo-boost": "^0.4.4",
     "graphql": "^14.5.6",
     "history": "^4.9.0",
+    "jest-styled-components": "^6.3.4",
```

Comment:
This seems to be a dependency only for tests, so should probably be a devDependency